make.ps1

```diff
@@ -0,0 +1,270 @@
+####### The starting point for the script is the bottom #######
+
+###############################################################
+########################## FUNCTIONS ##########################
+###############################################################
+function All-Command 
+{
+	$msBuild = FindMSBuild
+	$msBuildArguments = "/t:Rebuild /nr:false"
+	if ($msBuild -eq $null)
+	{
+		echo "Unable to locate an appropriate version of MSBuild."
+	}
+	else
+	{
+		$proc = Start-Process $msBuild $msBuildArguments -NoNewWindow -PassThru -Wait
+		if ($proc.ExitCode -ne 0)
+		{
+			echo "Build failed. If just the development tools failed to build, try installing Visual Studio. You may also still be able to run the game."
+		}
+		else
+		{
+			echo "Build succeeded."
+		}
+	}
+}
+
+function Clean-Command 
+{
+	$msBuild = FindMSBuild
+	$msBuildArguments = "/t:Clean /nr:false"
+	if ($msBuild -eq $null)
+	{
+		echo "Unable to locate an appropriate version of MSBuild."
+	}
+	else
+	{
+		$proc = Start-Process $msBuild $msBuildArguments -NoNewWindow -PassThru -Wait
+		rm *.dll
+		rm *.dll.config
+		rm mods/*/*.dll
+		rm *.pdb
+		rm mods/*/*.pdb
+		rm *.exe
+		rm ./*/bin -r
+		rm ./*/obj -r
+		echo "Clean complete."
+	}
+}
+
+function Version-Command 
+{
+	if ($command.Length -gt 1)
+	{
+		$version = $command[1]
+	}
+	elseif (Get-Command 'git' -ErrorAction SilentlyContinue)
+	{
+		$version = git name-rev --name-only --tags --no-undefined HEAD 2>$null
+		if ($version -eq $null)
```

Comment:
This doesn't appear to work:
![screen shot 2017-05-25 at 22 11 54](https://cloud.githubusercontent.com/assets/167819/26470972/8990bfdc-4197-11e7-8fbb-ce8368acde0b.png)